packages/@aws-cdk/aws-events/test/archive.test.ts

```diff
@@ -77,4 +77,20 @@ describe('archive', () => {
       },
     });
   });
+
+  test('should have defined defaultChild', () => {
+    const stack = new Stack();
+
+    const eventBus = new EventBus(stack, 'Bus');
+
+    const archive = new Archive(stack, 'Archive', {
+      sourceEventBus: eventBus,
+      eventPattern: {
+        account: [stack.account],
+      },
+      retention: Duration.days(10),
+    });
+
+    expect(archive.node.defaultChild).toBeDefined();
```

Comment:
I think this should test this against what we expect the value to be, not just for it to be defined or not.

Reply:
I have now changed the test. I'm not sure if it is the best way..